src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setVariable] = useState('');
+
+  const doesContain = (input: string) => (
+    input.toLowerCase().includes(query.toLowerCase())
```

Comment:
```suggestion
    input.toLowerCase().includes(query.toLowerCase().trim())
```
<img width="875" alt="Знімок екрана 2022-11-24 о 12 30 41" src="https://user-images.githubusercontent.com/106310796/203783732-c2d8ea98-38be-47af-8c84-6698b370f2e3.png">

Reply:
Yeah, U'r right) but it works like this `input.toLowerCase().trim().includes(query.toLowerCase())` )))